products/libllbuild/BuildSystem-C-API.cpp

```diff
@@ -309,8 +309,11 @@ class CAPIBuildSystemFrontendDelegate : public BuildSystemFrontendDelegate {
   virtual void commandCannotBuildOutputDueToMissingInputs(Command* command,
                Node* outputNode, SmallPtrSet<Node*, 1> inputNodes) override {
     if (cAPIDelegate.command_cannot_build_output_due_to_missing_inputs) {
-      auto str = outputNode->getName().str();
-      auto output = new CAPIBuildKey(BuildKey::makeNode(str));
+      CAPIBuildKey *output = nullptr;
```

Comment:
All the changes outside of ExternalCommand.cpp are to account for the fact that the outputs list of a dynamic task will be empty